src/main/java/seedu/address/logic/commands/LoginCommand.java

```diff
@@ -39,4 +60,36 @@ public CommandsParser getParser() throws CommandException {
     public CommandResult execute(Model model, CommandHistory history) {
         return new CommandResult(String.format(MESSAGE_SUCCESS, loginIdentity));
     }
+
+    /**
+     *
+     * @param loginIdentity input
+     * @return true or false depending if the email is present in the addressbook
+     */
+    private boolean isEmailPresent(String loginIdentity) {
+        return model.hasEmail(new Email(loginIdentity));
+    }
+
+    /**
+     *
+     * @param loginIdentity input
+     * @return true or false depending if the login identity is manager
+     */
+    private boolean isEmailManager(String loginIdentity) {
+        return model.getPerson(new Email(loginIdentity)).get()
```

Comment:
Would suggest breaking the code into different sections for better clarity and code quality!